### PR TITLE
WIP: Ways to give+faq

### PR DESCRIPTION
**Pull request by alanmoo**

### CODE REVIEW

**Comment by alanmoo** [comment]
This may not get merged but it might be useful for reference.

File: locales/en-US/messages.properties [hunk 1]
```diff
@@ -99,10 +99,10 @@ faq_intro=Below are the most frequently asked questions about donating to the Mo
...
+faq_item_3_paragraph=We understand that your privacy is very important. We ask for a minimum amount of information required to process credit card donations, including billing addresses. This allows our payment processor to verify your identity, process your payment, and prevent fraudulent charges to your credit card. We keep your information private -- if you have questions, please refer to our <a href="https://www.mozilla.org/privacy/principles/">Privacy Policy</a>. If you would rather not fill in your information on our online donation form, you can <a href="donate.mozilla.org/ways-to-give">mail a check</a> or donate directly <a href="https://sendto.mozilla.org/page/contribute/Give-Now">via Paypal</a>.
```
Comment (TheoChevalier) [inline locales/en-US/messages.properties]: <a href="donate.mozilla.org/ways-to-give"> should be <a href="https://donate.mozilla.org/ways-to-give">

Comment (ScottDowne) [inline locales/en-US/messages.properties]: Should be done.

**Comment by ScottDowne** [comment]
Bit of git nonsense, and ended up with another pr https://github.com/mozilla/donate.mozilla.org/pull/1663

**Pull request closed without merging**